dashboard/test/controllers/dynamic_config_controller_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+
+class DynamicConfigControllerTest < ActionController::TestCase
+  test 'inaccessible by non-admins' do
+    get :show
```

Comment:
This could help adding/removing actions easier

```
    non_logged_in_actions = %i[show gatekeeper_show gatekeeper_set gatekeeper_delete]
    non_logged_in_actions.each do |action|
      get action
      assert_redirected_to :new_user_session
    end
```